webapp/pages/index.vue

```diff
@@ -50,7 +50,7 @@
       :infinite-scroll-throttle-delay="800"
       :infinite-scroll-immediate-check="true"
     >
-      <hc-load-more v-if="true" :loading="$apollo.loading" @click="showMoreContributions" />
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-10-03T21:36:02Z" title="Thursday, October 3rd 2019, 11:36:02 pm +02:00">Oct 3, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -50,7 +50,7 @@
       :infinite-scroll-throttle-delay="800"
       :infinite-scroll-immediate-check="true"
     >
-      <hc-load-more v-if="true" :loading="$apollo.loading" @click="showMoreContributions" />
```


What the heck? :astonished: :laughing:

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-10-04T08:06:52Z" title="Friday, October 4th 2019, 10:06:52 am +02:00">Oct 4, 2019</time>_

---


yeah, hehe `v-if="true"`

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-10-11T06:41:49Z" title="Friday, October 11th 2019, 8:41:49 am +02:00">Oct 11, 2019</time>_

---


Oh my good 😬